### PR TITLE
feature: add baudrate to actisense configuration

### PR DESCRIPTION
**Pull request by e-sailing**
at the moment only standard baudrate 115200 is supported.

### CODE REVIEW

**Comment by mairas** [comment]
I'd actually prefer the term "bit rate" instead of "baud rate". The former is more accurate and SK as a project has sufficiently many touch points to communications technology that we might want to keep the terminology straight.

**Comment by e-sailing** [comment]
Well baud rate and bit rate are identical with respect to the serial interface. Baud rate and bit are among the accepted non-SI units.
But if we look deeper, the term baud rate or bit rate is not always correct. In mcu datasheet is only spoken by baud rate. This is defined by the frequency with which the data bits are sent. However, this so-called baud rate does not always correspond to the definition of baud rate or bit rate.
Example: so-called baud rate 9600
with the settings 8 data bits, 1 parity bit, 1.5 stop bits has a real baud rate or bit rate of 9142,857.

The pull request refers to the input of communication parameters for hardware. These should match the instructions in the manual so as not to confuse the user unnecessarily. Actisense uses the term baud rate in both the manual and the setup software.

**Comment by tkurki** [comment]
Both arguments have merit but this being the user interface I would be willing to pick _These should match the instructions in the manual so as not to confuse the user unnecessarily_.

But I'll let @sbender9 do the call, as I think this is more his domain.

File: providers/simple.js [hunk 1]
```diff
@@ -137,6 +137,7 @@ function nmea2000input (subOptions, logging) {
     return [
       new require('./actisense-serial')({
         device: subOptions.device,
+		baudrate: subOptions.baudrate,
```
Comment (tkurki) [inline providers/simple.js:140]: please fix Indentation

**Comment by e-sailing** [comment]
fixed Identation #677

**Comment by tkurki** [comment]
#669

**Pull request closed without merging**